chain-signatures/node/src/protocol/message.rs

```diff
@@ -218,6 +218,7 @@ impl MessageHandler for ResharingState {
         let q = queue.resharing_bins.entry(self.old_epoch).or_default();
         let mut protocol = self.protocol.write().await;
         while let Some(msg) = q.pop_front() {
+            tracing::trace!("handling new resharing message");
```

Comment:
Instead of having a new log call per resharing message, we should just log how many resharing messages we have